backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunServiceImpl.java

```diff
@@ -2597,4 +2598,25 @@ public void getExperimentRunsByDatasetVersionId(
           responseObserver, e, GetExperimentRunsByDatasetVersionId.Response.getDefaultInstance());
     }
   }
+
+  @Override
+  public void cloneExperimentRun(
```

Comment:
this needs protos changes right?

Reply:
Yes, it is there. https://github.com/VertaAI/modeldb/pull/1538